scripts/setup/terminate-psql-sessions

```diff
@@ -11,10 +11,11 @@ cd /
 username=$1
 
 shift
-tables=$(echo "'$*'" | sed "s/ /','/g")
+tables="$(printf "'%s'," "${@//\'/\'\'}")"
+tables="${tables%,}"
 
 if [ "$EUID" -eq 0 ]; then
-    sudo -u "$DEFAULT_USER" sh -c "psql postgres '$DEFAULT_USER'" <<EOF
+    sudo -u "$DEFAULT_USER" psql postgres "$DEFAULT_USER" <<EOF
```

Comment:
(Was this branch supposed to use `$username` like the other branch does?)

Reply:
Maybe?  I think `DEFAULT_USER` and `$username` are the same everywhere we use this.